jobma/api.py

```diff
@@ -46,7 +50,10 @@ def create_interview(interview):
     })
     response.raise_for_status()
     result = response.json()
-    interview.interview_id = result["interview_id"]
-    interview.save_and_log(None)
+    if "interview_link" in result:
+        interview.interview_url = result["interview_link"]
+        interview.save_and_log(None)
+    else:
+        log.error("Interview link not found in payload %s", result)
```

Comment:
Tiny nit: add a colon/dash/etc before the payload - `log.error("Interview link not found in payload - %s", result)`